pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -315,8 +315,9 @@ func (m *kubeGenericRuntimeManager) startContainer(ctx context.Context, podSandb
 		if handlerErr != nil {
 			klog.ErrorS(handlerErr, "Failed to execute PostStartHook", "pod", klog.KObj(pod),
 				"podUID", pod.UID, "containerName", container.Name, "containerID", kubeContainerID.String())
+			eventMessage := fmt.Sprintf("PostStartHook failed for container %s", container.Name)
 			// do not record the message in the event so that secrets won't leak from the server.
-			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, "PostStartHook failed")
+			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, eventMessage)
```

Comment:
nit: `recordContainerEvent` can format messages:
```suggestion
			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, "PostStartHook failed for container %s", container.Name)
```

Reply:
done